### DISCUSSION TITLE
Cool to call this the replacement for gnome-github-notifications?

### DISCUSSION THREAD

**Discussion by alexduf** [mackdk/gnome-github-manager#12] [Q&A] [answered]
Hey thanks for your work here, I'm the maintainer of https://github.com/alexduf/gnome-github-notifications and my priorities have changed a little bit so I can no longer maintain it.

I'm seeing you're actively maintaining this fork, would it be okay if I updated my readme to redirect to your repo? also do you have any plans to push your extension to extensions.gnome.org?

**Comment by mackdk** [comment, marked as answer]
Hi there!

Short answer is yes, but as I said in the other ticket, this is mostly something I started to learn Typescript and, while it's working, it's not yet really ready to be used by another user.

I still have things to do (add a proper README with building instruction, documentation, GNOME 44 support, a way to create the package automatically and so on) before I can submit it to `extensions.gnome.org`. As I working on this in my free time only, I don't have any time frame for when it can happen.
